source/engine/ui/editor/impl/EditorImpl.cpp

```diff
@@ -0,0 +1,39 @@
+#include "ui/editor/Editor.h"
```

Comment:
Should have git mv'd. This file is exactly Editor.cpp, but now has a factory function.